packages/cli/internal/pkg/cli/spec/project_test.go

```diff
@@ -82,11 +84,13 @@ schemaVersion: 0
 				},
 				Contexts: map[string]Context{
 					"ctx1": {
+						MaxVCpus: 256,
 						Engines: []Engine{
 							{Type: "wdl", Engine: "miniwdl"},
 						},
 					},
 					"ctx2": {
+						MaxVCpus: 256,
```

Comment:
Can you test the default case here?

Reply:
Yeah, I need to add an additional integ test that specifically validates defaults, since this test both marshals and unmarshals, it expects the input object and yaml string to be identical (in the default case they won't be, the input object contains the expected default value, and the yaml string contains no value)